sky/provision/kubernetes/utils.py

```diff
@@ -2023,3 +2026,112 @@ def get_skypilot_pods(context: Optional[str] = None) -> List[Any]:
             'kubectl get pods --selector=skypilot-cluster --all-namespaces'
         ) from None
     return pods
+
+
+@dataclasses.dataclass
+class KubernetesClusterInfo:
```

Comment:
How about renaming it to:
```suggestion
class KubernetesSkyPilotClusterInfo:
```
Reason: Kubenetes cluster is a bit confusing with kubernetes' term.

Reply:
good point, I was also thinking about this